prow/monitoring/grafana.ini

```diff
@@ -0,0 +1,15 @@
+[auth]
+disable_login_form = true
+disable_signout_menu = true
+[auth.basic]
+enabled = false
+[auth.anonymous]
+enabled = true
+org_role = Viewer
```

Comment:
Disabling all the auth mechanisms doesnt seem wise. Does this allow anyone to edit the dashboard?

Reply:
No:

    View any dashboard. This can be disabled on specific folders and dashboards.
    Cannot create or edit dashboards nor data sources.

https://grafana.com/docs/permissions/organization_roles/#viewer-role